util/reggen/README.md

```diff
@@ -96,6 +96,7 @@ name | required | string | name of the component
 clocking | required | list | clocking for the device
 bus_interfaces | required | list | bus interfaces for the device
 registers | required | list | list of register definition groups and offset control groups
+human_name | optional | string | human-readable name of the component
```

Comment:
Where is this generated from? If you just added this into the readme, maybe mention the human readable name in the commit message.

Reply:
These are generated using the command in the `util/selfdoc.py reggen` command. This optional field was added in [util/reggen/ip_block.py](https://github.com/lowRISC/opentitan/commit/6211349dbe2bde6626b19a4c44ed4f1c4d8a6896#diff-eab0f969150e7b8625e6753643a528484f1a6f07b71c2669e48e0f247e71d726).

I hadn't set up the CI check though, so the committer wasn't asked and told how to update the docs.

Reply:
Ok, thanks